pkg/helpers/identifier_string_parser_test.go

```diff
@@ -91,4 +94,251 @@ func Test_ParseIdentifierString(t *testing.T) {
 		require.NoError(t, err)
 		containsAll(t, parts, expected)
 	})
+
+	t.Run("empty identifier", func(t *testing.T) {
+		input := `""`
+		expected := []string{""}
+
+		parts, err := ParseIdentifierString(input)
+
+		require.NoError(t, err)
+		containsAll(t, parts, expected)
+	})
+
+	t.Run("handled correctly double quotes", func(t *testing.T) {
+		input := `""."."".".".""."".""."".""."".""."""""`
+		expected := []string{"", `.".`, `.".".".".".".".""`}
+
+		parts, err := ParseIdentifierString(input)
+
+		require.NoError(t, err)
+		containsAll(t, parts, expected)
+	})
+}
+
+func Test_ParseAccountObjectIdentifier(t *testing.T) {
+	testCases := []struct {
+		Input    string
+		Expected sdk.AccountObjectIdentifier
+		Error    string
+	}{
+		{Input: ``, Error: "incompatible identifier: "},
```

Comment:
I don't get it. It uses ParseIdentifierString, so this one has the same capabilities as ParseIdentifierString. It will error on \`\`, but won't on \`""\` (just as ParseIdentifierString)